interface/patient_file/encounter/forms.php

```diff
@@ -91,6 +91,12 @@
 <script type="text/javascript">
 $.noConflict();
 jQuery(document).ready( function($) {
+    <?php if ((boolean) $GLOBALS['expand_form']) { ?>
```

Comment:
@sjpadgett , Thanks for fixing this. We should prob make expand_form default to on in globals.inc.php.

Reply:
oh wait, disregard that, I didn't realize this was rel-502

Reply:
Looks like it is fixed in 5.0.3 but i've not looked at what it defaults to.